website/docs/design-patterns/layered-stack-configuration.md

```diff
@@ -0,0 +1,328 @@
+---
+title: Layered Stack Configuration Atmos Design Pattern
+sidebar_position: 13
+sidebar_label: Layered Stack Configuration
+description: Layered Stack Configuration Atmos Design Pattern
+---
+
+# Layered Stack Configuration
+
+The **Layered Stack Configuration** Design Pattern describes the mechanism of grouping Atmos components by category or function,
+adding the groups of components to layers, and importing the layers into top-level Atmos stacks.
+
+Each layer imports or configures a set of related Atmos components. Each Atmos component belongs to just one layer.
+Each layer can be managed separately, possibly by different teams.
+
+<br/>
+
+:::note
+The **Layered Stack Configuration** Design Pattern is similar to the [Partial Stack Configuration](/design-patterns/partial-stack-configuration)
+pattern, but instead of splitting top-level Atmos stacks into smaller parts by category or function, the **Layered Stack Configuration** design
+pattern prescribes adding separate layers to group the related Atmos components, and then importing the layers into top-level Atmos stacks.
+:::
+
+## Applicability
+
+Use the **Layered Stack Configuration** pattern when:
+
+- You have many Atmos components, and you need to group the components by category or function
+
+- You want to split the components into layers. Each layer should be managed and modified independent of the other layers, possibly by different
+  people or teams
+
+- You want to keep the configuration easy to manage and [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself)
+
+## Example
+
+In the following structure, we have various Terraform components (Terraform root modules) in the `components/terraform` folder.
+
+In the `stacks/catalog` folder, we define the defaults for each component using the [Component Catalog](/design-patterns/component-catalog) Atmos
+Design Pattern.
+
+In the `stacks/layers` folder, we define layers (groups of components), and import the related components into the layer manifests:
+
+- `load-balancers.yaml`
+- `data.yaml`
+- `dns.yaml`
+- `logs.yaml`
+- `notifications.yaml`
+- `firewalls.yaml`
+- `networking.yaml`
+- `eks.yaml`
+
```

Comment:
IMO, we should suggest a familiar model, similar to the OSI model. While not all layers, make sense it's a good mental model.
https://aws.amazon.com/what-is/osi-model/
![image](https://github.com/cloudposse/atmos/assets/52489/e10b1eda-ec84-4e8f-ad0c-eb36a7f02d61)

- `application.yaml`
- `platform.yaml`
- `network.yaml`
- `data.yaml`
- `compliance.yaml`
- `security.yaml`
- `observability.yaml`

IMO, 
- DNS is networking
- Logs and Notifications is observablity
- Load Balancers is networking
- Firewalls are security (or possibly networking)
- EKS is the Platform